src/harness/testcases/WINNF_FT_S_QPR_testcase.py

```diff
@@ -491,3 +494,342 @@ def test_WINNF_FT_S_QPR_8(self, config_filename):
+        'cbsdSerialNumber': device_2['cbsdSerialNumber'],
+        'airInterface': device_2['airInterface'],
+        'installationParam': device_2['installationParam'],
+        'measCapability': device_2['measCapability']
+    }
+    conditionals_3 = {
+        'cbsdCategory': device_3['cbsdCategory'],
+        'fccId': device_3['fccId'],
+        'cbsdSerialNumber': device_3['cbsdSerialNumber'],
+        'airInterface': device_3['airInterface'],
+        'installationParam': device_3['installationParam'],
+        'measCapability': device_3['measCapability']
+    }
+    conditionals_4 = {
+        'cbsdCategory': device_4['cbsdCategory'],
+        'fccId': device_4['fccId'],
+        'cbsdSerialNumber': device_4['cbsdSerialNumber'],
+        'airInterface': device_4['airInterface'],
+        'installationParam': device_4['installationParam'],
+        'measCapability': device_4['measCapability']
+    }
+    conditionals_5 = {
+        'cbsdCategory': device_5['cbsdCategory'],
+        'fccId': device_5['fccId'],
+        'cbsdSerialNumber': device_5['cbsdSerialNumber'],
+        'airInterface': device_5['airInterface'],
+        'installationParam': device_5['installationParam'],
+        'measCapability': device_5['measCapability']
+    }
+
+    # Remove conditionals from registration.
+    for device in [device_2, device_3, device_4, device_5]:
+      del device['cbsdCategory']
+      del device['airInterface']
+      del device['installationParam']
+      del device['measCapability']
+
+    # Grant Requests - N1
+    grant1N1 = {
+        'operationParam': {
+            'maxEirp': 19,
+            'operationFrequencyRange': {
+                'lowFrequency': 3650000000,
+                'highFrequency': 3660000000
+            }
+        }
+    }
+    grant2N1 = {
+        'operationParam': {
+            'maxEirp': 37,
+            'operationFrequencyRange': {
+                'lowFrequency': 3655000000,
+                'highFrequency': 3660000000
+            }
+        }
+    }
+    grant3N1 = {
+        'operationParam': {
+            'maxEirp': 35,
+            'operationFrequencyRange': {
+                'lowFrequency': 3650000000,
+                'highFrequency': 3660000000
+            }
+        }
+    }
+    grant4N1 = {
+        'operationParam': {
+            'maxEirp': 33,
+            'operationFrequencyRange': {
+                'lowFrequency': 3650000000,
+                'highFrequency': 3660000000
+            }
+        }
+    }
+    grant5N1 = {
+        'operationParam': {
+            'maxEirp': 30,
+            'operationFrequencyRange': {
+                'lowFrequency': 3650000000,
+                'highFrequency': 3670000000
+            }
+        }
+    }
+    # Grant Requests - N2
+    grant1N2 = {
+        'operationParam': {
+            'maxEirp': 19,
+            'operationFrequencyRange': {
+                'lowFrequency': 3650000000,
+                'highFrequency': 3660000000
+            }
```

Comment:
This grant will not be successful since grant1N1 already asked for the same frequency so when the CBSD tries to get a grant at the same frequency it'll get a 401 from the SAS (CBSD already has that grant).  Was this on purpose?

Reply:
Thanks for checking, it was a mistake. Fixed it.